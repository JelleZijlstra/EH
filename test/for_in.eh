#!/usr/bin/ehi
# For-in loops
arr = [1, 2]
for (k, v) in arr
	printvar k, v
end

arr2 = []
for (i, arr2->i) in arr
	printvar arr2
end

for i in 42..45
	printvar i
end
