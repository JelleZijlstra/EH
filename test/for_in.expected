%%stdout%%
@tuple <2> [
	@int 0
	@int 1
]
@tuple <2> [
	@int 1
	@int 2
]
@array [
	0 => @int 1
]
@array [
	0 => @int 1
	1 => @int 2
]
@int 42
@int 43
@int 44
@int 45
%%stderr%%
